README.md

```diff
@@ -61,10 +85,12 @@ function printDataToSign(bytes memory data) internal pure {
     console.log("^^^^^^^^");
 }
 ```
+
 ...where `data` is 66 bytes in length.
 
 Example output:
-```shell
+
+````shell
```

Comment:
any reason we added a ` here and below?

Reply:
@mdehoog 

Heya, thanks for pointing this out -- looks like the issue is that on `main` there are three extra backticks at the end of the `Signature` line at the EOF. My linter picked this up and thought it was supposed to be part of a fenced code block so it attempted to fix it by adding the extra backtick.

![2023-10-02 at 17 50 43@2x](https://github.com/base-org/eip712sign/assets/1130872/d82fca09-9486-42b8-a2cc-806c81605e0b)

Removed the three extra backticks from the Signature line to fix the issue and left the code snippet as-is otherwise. 👍

Reply:
oh cool good find, thanks for fixing